product_stock_location/models/stock_warehouse_orderpoint.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models, fields, api
+
+
+class StockWarehouseOrderpoint(models.Model):
+    _inherit = 'stock.warehouse.orderpoint'
+
+    @api.multi
+    @api.depends('product_stock_location_ids',
+                 'product_stock_location_ids.product_location_qty',
+                 'product_stock_location_ids.incoming_location_qty',
+                 'product_stock_location_ids.outgoing_location_qty',
+                 'product_stock_location_ids.virtual_location_qty'
+                 )
+    def _compute_product_available_qty(self):
```

Comment:
@jbeficent any reasons you redefined the computed fields? You don't make any changes in the computation.